client/components/sidebar.vue

```diff
@@ -1,64 +1,67 @@
 <template>
-  <v-container fluid class="sidebar__users">
+  <v-container @mousedown.stop fluid class="sidebar__users">
```

Comment:
From my experience preventing propagation of events often backfires. For example, many times when you have some overlay (like dialog) you want it to be closed if you click outside of it. But if user clicks on element which is stopping propagation then this does not happen.

So here you are preventing propagation for whole sidebar. Is this really what you want?

And elsewhere in the app there is also `.stop` around. Is it really necessary? Isn't preventing default action often enough?